src/backend/marsha/core/views.py

```diff
@@ -502,6 +503,8 @@ def _get_app_data(self, lti):
             context={"request": self.request},
         ).data
 
+        playlist = PlaylistLiteSerializer(playlist).data
```

Comment:
reassigning a variable is a bad idea IMO. You can directly use `PlaylistLiteSerializer(playlist).data` line 536